site/sidebar.ts

```diff
@@ -131,14 +131,22 @@ export const sidebar = [
       {
         text: 'Wallet',
         items: [
+          {
+            text: 'WalletDropdownBaseName',
+            link: '/wallet/wallet-dropdown-base-name',
+          },
+          {
+            text: 'WalletDropdownDisconnect',
+            link: '/wallet/wallet-dropdown-disconnect',
+          },
+          {
+            text: 'WalletDropdownLink',
+            link: '/wallet/wallet-dropdown-link',
+          },
           {
```

Comment:
Let's have Wallet as the first component.

Reply:
Ok - And I will update `Identity` to also be the first component so every section matches. 
<img width="101" alt="Screenshot 2024-08-13 at 3 31 37 PM" src="https://github.com/user-attachments/assets/461d84e9-e7df-44de-b23a-a8e1f5e148f8">